components/x-interaction/__tests__/x-interaction.test.jsx

```diff
@@ -163,6 +201,25 @@ describe('x-interaction', () => {
 				target.find(Base).prop('bar')
 			).toBe(15);
 		});
+
+		it('should pass actions to actionsRef on mount and null on unmount', async () => {
+			const actionsRef = jest.fn();
+
+			const Base = () => null;
+			const Wrapped = withActions({
+				foo() {},
+			})(Base);
+
+			const target = mount(<Wrapped actionsRef={actionsRef} />);
+
+			expect(actionsRef).toHaveBeenCalled();
+			expect(actionsRef.mock.calls[0][0]).toHaveProperty('foo');
+
+			// TODO: target.unmount() doesn't actually unmount the child for some reason?
+			target.find('InteractionClass').instance().componentWillUnount();
```

Comment:
`componentWillUnount` -> `componentWillUnmount`

Reply:
welp, that would explain why it wasn't being called on `unmount`